### PR TITLE
Update memorandum.md

### PR DESCRIPTION
**Pull request by puchojenso**

### CODE REVIEW

File: memorandum.md [hunk 1]
```diff
@@ -4,32 +4,26 @@ layout: page
 permalink: /memorandum
 ---
 
-# Memorandum
-BlackPool aims to become a leading provider of financial derivatives in digital sports marketplaces, including asset valuation indexes, leasing, insurance mechanisms and more, starting with SoRare, the leading blockchain footfall fantasy game.
+BlackPool is the first decentralised autonomous organisation (DAO) built solely for digital sport trading.
```
Comment (QMilne) [inline memorandum.md:7]: NFT? Not sure we can label Axie as sport?

File: memorandum.md [hunk 2]
```diff
@@ -4,32 +4,26 @@ layout: page
...
+This combination of skills and knowledge means BlackPool will be able to track live data from prominent sports journalists, and use information on injuries, contract extensions and player performance to our advantage.
 
-<br />
-• to improve the economic and financial capabilities of the clubs, increasing their transparency and credibility;
-• to place the necessary importance on the protection of creditors and to ensure that clubs settle their liabilities with employees, social/tax authorities and other clubs punctually;
-• to introduce more discipline and rationality in club football finances;
-• to encourage clubs to operate on the basis of their own revenues;
-• to encourage responsible spending for the long-term benefit of football;
-• to protect the long-term viability and sustainability of the game's stakeholders.
+BlackPool’s vision is to provide value to SoRare and therefore will not play against any potential Decentralised Financial Fair Play Regulations (DFFPR).
```
Comment (QMilne) [inline memorandum.md:26]: Would remove that altogether (the fair play part) and instead extend on the other games we have our eyes on (Axie, God Unchained, the NBA game at some point?)

File: memorandum.md (hunk 1 above)
Comment (bneiluj) [reply memorandum.md:7]: agreed

File: memorandum.md (hunk 2 above)
Comment (bneiluj) [reply memorandum.md:26]: agreed

**Pull request merged**